roles/openshift_sdn/files/sdn.yaml

```diff
@@ -77,16 +77,24 @@ spec:
             fi
           done
 
+          #BUG 1728081: If /etc/sysconfig/origin-node got mounted as DirectoryOrCreate; clean it up so we can ultimately mount /etc/sysconfig/origin-node as FileOrCreate
+          if [[ -d /etc/sysconfig/origin-node ]]; then
+            echo "Deleting the origin-node dir"
+            rmdir /etc/sysconfig/origin-node
```

Comment:
can you do `rmdir ... || true` - if this somehow fails, we shouldn't take networking down on the node.